lib/generators/shopify_app/rotate_shopify_token_job/rotate_shopify_token_job_generator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails/generators/base'
+
+module ShopifyApp
+  module Generators
+    class RotateShopifyTokenJobGenerator < Rails::Generators::Base
+      source_root File.expand_path('../templates', __FILE__)
+
+      def add_rotate_shopify_token_job
+        copy_file('rotate_shopify_token_job.rb', "app/jobs/shopify/rotate_shopify_token_job.rb")
```

Comment:
I wonder if they should they be `app/jobs/shopify_app/...`, but the way they are now is the pattern though.

Reply:
I think the `shopify` namespace makes enough sense, since the job deals with Shopify concerns 👍

Reply:
I think it's okay to maintain the existing pattern.